### PR TITLE
Turned bootstrap API only + new versions

### PR DESCRIPTION
**Pull request by icoluccio**
## Summary
Removed all non-API stuff including webpacker and upgraded most versions.

### CODE REVIEW

File: public/assets/.sprockets-manifest-23f99e3321a478c83f20330bc2f9bd87.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"files":{"application-4e458256976996c68fa5a7b429e2aee85328370e969f8e28819e480dbe6f2222.js":{"logical_path":"application.js","mtime":"2019-03-01T16:57:14-03:00","size":21491,"digest":"4e458256976996c68fa5a7b429e2aee85328370e969f8e28819e480dbe6f2222","integrity":"sha256-TkWCVpdplsaPpae0KeKu6FMoNw6Wn44ogZ5IDb5vIiI="},"application-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855.css":{"logical_path":"application.css","mtime":"2019-03-01T14:44:14-03:00","size":0,"digest":"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855","integrity":"sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU="}},"assets":{"application.js":"application-4e458256976996c68fa5a7b429e2aee85328370e969f8e28819e480dbe6f2222.js","application.css":"application-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855.css"}}
```
Comment (nicozar95) [inline public/assets/.sprockets-manifest-23f99e3321a478c83f20330bc2f9bd87.json]: Is this necesary?

File: Gemfile [hunk 2]
```diff
@@ -1,98 +1,48 @@
...
-gem 'foreman'
+gem 'devise_token_auth', '~> 1.0'
 
-# Active Admin
-gem 'activeadmin', git: 'https://github.com/activeadmin/activeadmin'
-gem 'inherited_resources', git: 'https://github.com/activeadmin/inherited_resources'
-
-# Enables Slim templates
-gem 'slim-rails'
+gem 'active_model_serializers', '~> 0.10'
```
Comment (nicozar95) [inline Gemfile]: Should we considered [Fast Json API](https://github.com/Netflix/fast_jsonapi)? It's really simillar to `ActiveModel::Serializers` and a little more performant.

File: public/assets/.sprockets-manifest-23f99e3321a478c83f20330bc2f9bd87.json (hunk 1 above)
Comment (icoluccio) [reply public/assets/.sprockets-manifest-23f99e3321a478c83f20330bc2f9bd87.json]: Clearly not :D

File: Gemfile (hunk 2 above)
Comment (icoluccio) [reply Gemfile]: I think it's worth the research but since most Wolox projects use active_model_serializers I'll keep it in this PR

**Comment by icoluccio** [comment]
Don't merge, amend was wrong

File: Gemfile (hunk 2 above)
Comment (fedeest) [reply Gemfile]: For what I've investigated on Fast Json API, it's not exactly the same. It actually has many problems (as in complications, not actual problems) to render nested models which make it more complicated to use.

I'd say that we eventually migrate towards it, but in the meantime we have to stick with ActiveModel Serializers till we fully understand FJAPI

File: Gemfile [hunk 3]
```diff
@@ -1,98 +1,49 @@
...
 gem 'rack-attack'
 
-gem 'webpacker', '~> 3.0'
-
 gem 'health_check', '~> 3.0'
 
 group :development do
   # Gem to detect N+1 queries
-  gem 'better_errors'
   gem 'bullet'
```
Comment (fedeest) [inline Gemfile]: Shouldn't this be in test too?

**Comment by fedeest** [comment]
Merging this anytime soon @icoluccio ?

File: config/locales/models/general/en.yml [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+      last_word_connector: ", and "
+      two_words_connector: " and "
+      words_connector: ", "
+  time:
+    am: am
+    formats:
+      default: "%a, %d %b %Y %H:%M:%S %z"
+      long: "%B %d, %Y %H:%M"
+      short: "%d %b %H:%M"
+    pm: pm
```
Comment (fedeest) [inline config/locales/models/general/en.yml]: Missing line break at EOF

File: config/locales/models/general/es.yml [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+      last_word_connector: " y "
+      two_words_connector: " y "
+      words_connector: ", "
+  time:
+    am: am
+    formats:
+      default: "%A, %-d de %B de %Y %H:%M:%S %z"
+      long: "%-d de %B de %Y %H:%M"
+      short: "%-d de %b %H:%M"
+    pm: pm
```
Comment (fedeest) [inline config/locales/models/general/es.yml]: Missing line break at EOF

File: Gemfile (hunk 2 above)
Comment (holywyvern) [reply Gemfile]: As tested in P&G I don't suggest fast json api for now

File: script/pre-push.sh [hunk 6]
```diff
@@ -10,26 +10,6 @@ then
...
-RUN_CHECK_CMD='yarn run lint'
-RUN_TESTS_OUTPUT=`${RUN_CHECK_CMD}`
-
-if [ $? -eq 1 ]
-then
-  echo "\033[34mCan't commit! You have eslint problems!!!\033[0m"
-  exit 1
-fi
-
 echo "\033[34mRunning Rubocop\033[0m"
```
Comment (lucasVoboril) [inline script/pre-push.sh]: We can also add brakeman in this script

```
echo "\033[34mRunning Brakeman\033[0m"
RUN_CHECK_CMD='bundle exec brakeman --exit-on-error --quiet'
RUN_TESTS_OUTPUT=`${RUN_CHECK_CMD}`

if [ $? -ne 0 ]
then
  echo "\033[34mCan't commit! You have security warnings!!!\033[0m"
  exit 1
fi

echo "\033[34mAll checks passed. You didn't break anything. Congrats!\n\033[0m"
exit 0
```

File: Gemfile [hunk 7]
```diff
@@ -1,98 +1,49 @@
 source 'https://rubygems.org'
```
Comment (lucasVoboril) [inline Gemfile]: We can also upgrade many gems, but it's ok

```
➜  rails-bootstrap git:(bootstrap-lite-test) bundle outdated       
Fetching gem metadata from https://rubygems.org/.......
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies.....

Outdated gems included in the bundle:
  * actioncable (newest 6.0.1, installed 5.1.6.1)
  * actionmailer (newest 6.0.1, installed 5.1.6.1)
  * actionpack (newest 6.0.1, installed 5.1.6.1)
  * actionview (newest 6.0.1, installed 5.1.6.1)
  * active_model_serializers (newest 0.10.10, installed 0.10.9, requested ~> 0.10) in groups "default"
  * activejob (newest 6.0.1, installed 5.1.6.1)
  * activemodel (newest 6.0.1, installed 5.1.6.1)
  * activerecord (newest 6.0.1, installed 5.1.6.1)
  * activesupport (newest 6.0.1, installed 5.1.6.1)
  * addressable (newest 2.7.0, installed 2.6.0)
  * arel (newest 9.0.0, installed 8.0.0)
  * bcrypt (newest 3.1.13, installed 3.1.12)
  * brakeman (newest 4.7.1, installed 4.4.0) in groups "development, test"
  * bullet (newest 6.0.2, installed 5.9.0) in groups "development"
  * byebug (newest 11.0.1, installed 11.0.0) in groups "development, test"
  * capybara (newest 3.29.0, installed 3.14.0) in groups "test"
  * codestats-metrics-reporter (newest 0.1.13, installed 0.1.9, requested = 0.1.9) in groups "test"
  * crass (newest 1.0.5, installed 1.0.4)
  * devise (newest 4.7.1, installed 4.5.0)
  * devise_token_auth (newest 1.1.3, installed 1.0.0, requested ~> 1.0) in groups "default"
  * docile (newest 1.3.2, installed 1.3.1)
  * dotenv (newest 2.7.5, installed 2.7.1)
  * dotenv-rails (newest 2.7.5, installed 2.7.1) in groups "development, test"
  * erubi (newest 1.9.0, installed 1.8.0)
  * excon (newest 0.68.0, installed 0.62.0)
  * factory_bot (newest 5.1.1, installed 5.0.2)
  * factory_bot_rails (newest 5.1.1, installed 5.0.1) in groups "development, test"
  * faker (newest 2.7.0, installed 1.9.3) in groups "development, test"
  * ffi (newest 1.11.1, installed 1.10.0)
  * flay (newest 2.12.1, installed 2.12.0)
  * flog (newest 4.6.3, installed 4.6.2)
  * fog-aws (newest 3.5.2, installed 3.4.0)
  * hashdiff (newest 1.0.0, installed 0.3.8)
  * hashie (newest 4.0.0, installed 3.6.0)
  * httparty (newest 0.17.1, installed 0.16.4)
  * i18n (newest 1.7.0, installed 1.6.0)
  * jaro_winkler (newest 1.5.4, installed 1.5.2)
  * jsonapi-renderer (newest 0.2.2, installed 0.2.0)
  * libv8 (newest 7.3.492.27.1, installed 3.16.14.19)
  * listen (newest 3.2.0, installed 3.1.5, requested ~> 3.1.5) in groups "development"
  * loofah (newest 2.3.1, installed 2.2.3)
  * mime-types (newest 3.3, installed 3.2.2)
  * mime-types-data (newest 3.2019.1009, installed 3.2018.0812)
  * mini_mime (newest 1.0.2, installed 1.0.1)
  * minitest (newest 5.13.0, installed 5.11.3)
  * multi_json (newest 1.14.1, installed 1.13.1)
  * nio4r (newest 2.5.2, installed 2.3.1)
  * nokogiri (newest 1.10.5, installed 1.10.1)
  * oga (newest 2.15, installed 1.3.1)
  * parallel (newest 1.18.0, installed 1.14.0)
  * parser (newest 2.6.5.0, installed 2.6.0.0)
  * path_expander (newest 1.1.0, installed 1.0.3)
  * public_suffix (newest 4.0.1, installed 3.0.3)
  * puma (newest 4.3.0, installed 3.12.0, requested ~> 3.12) in groups "default"
  * pundit (newest 2.1.0, installed 2.0.1) in groups "default"
  * rack (newest 2.0.7, installed 2.0.6)
  * rack-attack (newest 6.2.1, installed 5.4.2) in groups "default"
  * rack-cors (newest 1.0.3, installed 1.0.2, requested ~> 1.0.2) in groups "default"
  * rack-protection (newest 2.0.7, installed 2.0.5)
  * rails (newest 6.0.1, installed 5.1.6.1, requested ~> 5.1.6.1) in groups "default"
  * rails-html-sanitizer (newest 1.3.0, installed 1.0.4)
  * railties (newest 6.0.1, installed 5.1.6.1)
  * rake (newest 13.0.0, installed 12.3.2)
  * redis (newest 4.1.3, installed 4.1.0)
  * reek (newest 5.4.0, installed 5.3.1)
  * regexp_parser (newest 1.6.0, installed 1.3.0)
  * responders (newest 3.0.0, installed 2.4.1)
  * rollbar (newest 2.22.1, installed 2.19.2) in groups "default"
  * rspec-core (newest 3.9.0, installed 3.8.0)
  * rspec-expectations (newest 3.9.0, installed 3.8.2)
  * rspec-mocks (newest 3.9.0, installed 3.8.0) in groups "test"
  * rspec-rails (newest 3.9.0, installed 3.8.2) in groups "test"
  * rspec-support (newest 3.9.0, installed 3.8.0)
  * rubocop (newest 0.76.0, installed 0.65.0, requested ~> 0.65.0) in groups "development, test"
  * rubocop-rspec (newest 1.36.0, installed 1.32.0, requested ~> 1.32) in groups "development, test"
  * ruby-progressbar (newest 1.10.1, installed 1.10.0)
  * ruby_parser (newest 3.14.1, installed 3.12.0)
  * rubycritic (newest 4.2.1, installed 4.0.0) in groups "test"
  * sexp_processor (newest 4.13.0, installed 4.11.0)
  * shoulda-matchers (newest 4.1.2, installed 4.0.1) in groups "test"
  * sidekiq (newest 6.0.3, installed 5.2.5, requested ~> 5.2) in groups "default"
  * simplecov (newest 0.17.1, installed 0.16.1) in groups "test"
  * spring (newest 2.1.0, installed 2.0.2) in groups "development"
  * sprockets (newest 4.0.0, installed 3.7.2)
  * tty-which (newest 0.4.1, installed 0.4.0)
  * tzinfo (newest 2.0.0, installed 1.2.5)
  * unicode-display_width (newest 1.6.0, installed 1.4.1)
  * uniform_notifier (newest 1.13.0, installed 1.12.1)
  * webmock (newest 3.7.6, installed 3.5.1) in groups "test"
  * websocket-driver (newest 0.7.1, installed 0.6.5)
  * websocket-extensions (newest 0.1.4, installed 0.1.3)
```

Comment (lucasVoboril) [reply Gemfile]: :eyes: if you delete Gemfile.lock and then run bundle, it upgrade many versions 

```
➜  rails-bootstrap git:(bootstrap-lite-test) ✗ bundle outdated
Fetching gem metadata from https://rubygems.org/.......
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies.....

Outdated gems included in the bundle:
  * actioncable (newest 6.0.1, installed 5.1.6.2)
  * actionmailer (newest 6.0.1, installed 5.1.6.2)
  * actionpack (newest 6.0.1, installed 5.1.6.2)
  * actionview (newest 6.0.1, installed 5.1.6.2)
  * activejob (newest 6.0.1, installed 5.1.6.2)
  * activemodel (newest 6.0.1, installed 5.1.6.2)
  * activerecord (newest 6.0.1, installed 5.1.6.2)
  * activesupport (newest 6.0.1, installed 5.1.6.2)
  * arel (newest 9.0.0, installed 8.0.0)
  * codestats-metrics-reporter (newest 0.1.13, installed 0.1.9, requested = 0.1.9) in groups "test"
  * hashie (newest 4.0.0, installed 3.6.0)
  * libv8 (newest 7.3.492.27.1, installed 3.16.14.19)
  * listen (newest 3.2.0, installed 3.1.5, requested ~> 3.1.5) in groups "development"
  * oga (newest 2.15, installed 1.3.1)
  * puma (newest 4.3.0, installed 3.12.1, requested ~> 3.12) in groups "default"
  * rails (newest 6.0.1, installed 5.1.6.2, requested ~> 5.1.6.1) in groups "default"
  * railties (newest 6.0.1, installed 5.1.6.2)
  * rubocop (newest 0.76.0, installed 0.65.0, requested ~> 0.65.0) in groups "development, test"
  * rubocop-rspec (newest 1.36.0, installed 1.35.0, requested ~> 1.32) in groups "development, test"
  * sidekiq (newest 6.0.3, installed 5.2.7, requested ~> 5.2) in groups "default"
  * tzinfo (newest 2.0.0, installed 1.2.5)
  * unicode-display_width (newest 1.6.0, installed 1.4.1)
  * websocket-driver (newest 0.7.1, installed 0.6.5)
```

File: README.md [hunk 8]
```diff
@@ -29,16 +29,7 @@ You can skip the hook by adding `--no-verify` to your `git push`.
...
-
-```bash
-  curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-  echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-  sudo apt remove cmdtest # Some Ubuntu systems comes with cmdtest installed by default.
-  sudo apt-get update && sudo apt-get install yarn
-```
-
-### 3- Installing Rails gems
+### 2- Installing Rails gems
```
Comment (lucasVoboril) [inline README.md]: Can we delete this incorrect badge?
```
[![Codestats](http://codestats-url/organizations/your-organization/projects/rails-bootstrap/badge)](http://codestats-url/organizations/your-organization/projects/rails-bootstrap/badge)
```

Comment (icoluccio) [reply README.md]: I'm not sure what we should do with Codestats in general though

File: Gemfile (hunk 7 above)
Comment (icoluccio) [reply Gemfile]: We did that for #83!

File: script/pre-push.sh (hunk 6 above)
Comment (icoluccio) [reply script/pre-push.sh]: Sounds fine to me but we should do that in another PR so this actually gets merged 👀 wanna add that issue?

**Pull request merged**